recipes/lz4/lz4_r131.oe

```diff
@@ -0,0 +1,5 @@
+require ${PN}.inc
+
+SRC_URI += "file://windows-h-include.patch"
+SRC_URI += "file://exeext.patch"
```

Comment:
I don't understand your con reasoning :) if you don't build for mingw, and those are applied conditionally, how would that couse a patch conflict? If I added another patch only for Linux builds?